### ISSUE TITLE
Setting Manual IDs on a Repeater field ignores post types

### ISSUE THREAD

**Issue by ukdesignservices** [soflyy/oxygen-bugs-and-features#2179]
**Describe the bug**
Setting Manual IDs on a Repeater field removes any Post Types from the query. Regardless of what you add in the Post Type field, the actual query runs with 'post' only in the query.

**A link to a Sandbox site where the bug has been reproduced https://oxygenbuilder.com/try? (REQUIRED)**
https://oxygen-qa9thxfkwjvfl.oxygen-demo.qsandbox.me/sample-page/
The page is displaying blank because of the issue described. 

The exact problem can be seen by looking at the repeater parameters, and the results from Query Monitor:

Repeater Parameters: https://www.dropbox.com/s/jrma0f2afyd8jys/repeater-params.PNG?dl=0
Repeater Query: https://www.dropbox.com/s/0hoi7szm369ntj1/repeater-query.PNG?dl=0

**Comment by ukdesignservices** [comment]
This issue can be traced back to the following code:

**oxygen/component-framework/components/classes/dynamic-list.class.php @ _line 869_**

```
// post type
if ($options['query_post_ids']) {
    $args['post__in'] = explode(",",$options['query_post_ids']);
}
else {
    $args['post_type'] = $options['query_post_types'];
}
```

Post Types can only be set if there are no Post IDs specified.

**Comment by Spellhammer** [comment]
Fixed in Oxygen 4.1 beta 1.
